reference/5.1/PowershellGet/New-ScriptFileInfo.md

```diff
@@ -16,31 +16,42 @@ Creates a script file with metadata.
 
 ## SYNTAX
 
+### All
+
 ```
 New-ScriptFileInfo [[-Path] <String>] [-Version <String>] [-Author <String>] -Description <String>
  [-Guid <Guid>] [-CompanyName <String>] [-Copyright <String>] [-RequiredModules <Object[]>]
  [-ExternalModuleDependencies <String[]>] [-RequiredScripts <String[]>]
  [-ExternalScriptDependencies <String[]>] [-Tags <String[]>] [-ProjectUri <Uri>] [-LicenseUri <Uri>]
- [-IconUri <Uri>] [-ReleaseNotes <String[]>] [-PrivateData <String>] [-PassThru] [-Force] [-WhatIf] [-Confirm]
- [<CommonParameters>]
+ [-IconUri <Uri>] [-ReleaseNotes <String[]>] [-PrivateData <String>] [-PassThru] [-Force]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **New-ScriptFileInfo** cmdlet creates a PowerShell script file, including metadata about the script.
+
+The `New-ScriptFileInfo` cmdlet creates a PowerShell script file, including metadata about the
+script.
 
 ## EXAMPLES
 
 ### Example 1: Create a script file and specify its version, author, and description
+
+In this example, a script file is created and its contents are displayed in the PowerShell console.
+
+```powershell
+New-ScriptFileInfo -Path C:\Test\Temp-Scriptfile.ps1 -Version 1.0 -Author "pattif@contoso.com" `
```

Comment:
Using splatting instead of line-continuation.

Reply:
Updated doc to use splatting.